src/cmd_line/commands/file.ts

```diff
@@ -23,13 +29,48 @@ export class FileCommand extends node.CommandBase {
         return this._arguments;
     }
 
+    getViewColumn(sideBySide: boolean) : vscode.ViewColumn {
+        const active = vscode.window.activeTextEditor;
+        if (!active) {
+            return vscode.ViewColumn.One;
+        }
+
+        if (!sideBySide) {
+            return active.viewColumn;
+        }
+
+        switch (active.viewColumn) {
+            case vscode.ViewColumn.One:
```

Comment:
Ok, I understand now. 

> If we separate it to two functions, we may need move logics to the caller so I'd like to keep it this way.

I don't think so. The caller still needs to determine whether to set `sideBySide` to be `true` or `false`. That's the same as determining whether to call getActiveViewColumn() and getViewColumnToRight(). I suggest splitting the function since those feel to me like separate bits of functionality.